Normalize spec shortnames

TrisTOON opened the pull request:
This PR (part of #197) updates spec shortnames, i.e. keys in the `Specs` object, to always match the `dev` link when it exists.

I also updated some links to includes the level of the module and added the "tr" links for *Media Queries Level 5*, *Transforms Level 2* and *Box Sizing Level 4*.

Zefling commented on tests.js:
```diff
@@ -3886,7 +3888,7 @@ window.Specs = {
 		"links": {
-			"tr": "pointerevents2",
+			"tr": "pointerevents",
 			"dev": "pointerevents",
```
Pointer Events Level 2

TR : https://w3c.github.io/pointerevents/
DEV : https://www.w3.org/TR/pointerevents2/

TrisTOON replied:
I just saw that the draft is already at level 3 and that https://www.w3.org/TR/pointerevents3/ is a thing.
Should i link to `pointerevents3` instead?

Zefling replied:
Level 1 exist too: https://www.w3.org/TR/pointerevents1/
We take the functionalities in the oldest version.

Zefling commented on tests.js:
```diff
@@ -1223,6 +1223,7 @@ window.Specs = {
 	"mediaqueries-5": {
 		"title": "Media Queries Level 5",
 		"links": {
+			"tr": "mediaqueries-5",
```
Currently not existing

TrisTOON replied:
It's there, though :thinking: : https://www.w3.org/TR/mediaqueries-5/

Zefling replied:
Ho sorry.

LeaVerou commented:
@TrisTOON since you're contributing so much, would you be ok with being added to the repo as a collaborator?

TrisTOON commented:
> @TrisTOON since you're contributing so much, would you be ok with being added to the repo as a collaborator?

I would be delighted, it means a lot to me. :smile:

TrisTOON commented:
I rebased the PR and changed the `tr` link of the *Pointer Events* spec to the first level, since all keywords currently tested in the `touch-action` property were actually introduced right from this level.
